services/storage-users/README.md

```diff
@@ -0,0 +1,80 @@
+# Storage-Users Service
+
+Purpose and description to be added
+
+## CLI Commands
+
+### Manage Unfinished Uploads
+
+<!-- referencing: [oCIS FS] clean up aborted uploads https://github.com/owncloud/ocis/issues/2622 -->
+
+When using Infinite Scale as user storage, a directory named `storage/users/uploads` can be found in the Infinite Scale data folder. This is an intermediate directory based on [TUS](https://tus.io) which is an open protocol for resumable uploads. Each upload consists of a _blob_ and a _blob.info_ file. Note that the term _blob_ is just a placeholder.
+
+*   **If an upload succeeds**, the _blob_ file will be moved to the target and the _blob.info_ file will be deleted.
+
+*   **In case of incomplete uploads**, the _blob_ and _blob.info_ files will continue to recieve data until either the upload succeeds in time or the upload expires based on the `STORAGE_USERS_UPLOAD_EXPIRATION` variable, see the table below for details.
```

Comment:
```suggestion
*   **In case of incomplete uploads**, the _blob_ and _blob.info_ files will continue to receive data until either the upload succeeds in time or the upload expires based on the `STORAGE_USERS_UPLOAD_EXPIRATION` variable, see the table below for details.
```